wordpress-mobile/WordPress-iOS
# ULS Password: update Auth to use 1Password button action

Ref: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/issues/352
Auth PR: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/377

This uses the Auth changes that enables the 1Password button on the unified Password view.

**Prerequisites:**
- Enable the `unifiedGoogle` feature flag.
- You'll need a device configured with 1Password. As noted in the Auth PR, the iOS version check is temporarily disabled so you can test with an iOS13+ device.
- A gmail account that:
  - Has a WP password set.
  - Is _not_ connected to Google. (https://wordpress.com/me/security/social-login)

<img width="500" alt="Screen Shot 2020-08-11 at 1 59 25 PM" src="https://user-images.githubusercontent.com/1816888/89943226-1bdcd700-dbdb-11ea-955d-af855fdfef76.png">

**To test:**

- Go to Log In > Continue with Google.
- Select an account and complete Google auth.
- On the password entry view, tap the 1Password button on email field.

<kbd>![1password_button](https://user-images.githubusercontent.com/1816888/90295071-3d40fb80-de45-11ea-949b-a20daee5f8bb.jpg)</kbd>

- In 1Password, select an account.
- Verify:
  - The displayed email address _does not_ change.
  - The password field is populated.
  - You can log in.

PR submission checklist:
